content/api/apps/validation/_index.en.md

```diff
@@ -74,8 +74,8 @@ Example response:
 The application will provide a method to validate the datamodel without creating a instance of the data. Data must be provided as formdata.
 The validate method takes a data file of an dataType and performs validation on that file. It returns a validation report.
 
-![Flowchart for data validation](data-validate.png "Validate data")
+![Flowchart for data validation](/content/api/data-validate.png "Validate data")
```

Comment:
This will not work. Only the content of the `content` folder is part of the URL after the site is built.

The real problem here is that `data-validate.png` is located directly under the `/content/api`-folder. If you just move it to the `/content/api/apps/validatation`-folder instead, it should work. This image not used from any other page, so should not break anything.

Tip: If you [run docs.altinn.studio locally](https://github.com/Altinn/altinn-studio-docs#running-altinn-studio-docs-locally) you can be 100% certain that your PRs are working 👍

Reply:
The github gui told me that it would work :(